libs/xb/op.ml

```diff
@@ -31,13 +31,13 @@ let operation_c_mapping =
            Transaction_end; Introduce; Release;
            Getdomainpath; Write; Mkdir; Rm;
            Setperms; Watchevent; Error; Isintroduced;
-           Resume; Set_target |]
+           Resume; Set_target; |]
```

Comment:
I might have put XSReset_watches and XSInvalid  up in that list first, using a semi-colon to separate them, before I realized they belonged down there.  I just forgot to remove the semi-colon too.  Will fix.